### DISCUSSION TITLE
What happened to the Rocky Linux LXC scripts?

### DISCUSSION THREAD

**Discussion by amlucent** [tteck/Proxmox#2376] [Q&A] [answered]
What happened to the Rocky Linux LXC scripts? I mean I was using them but I understand if you decided they werent popular enough to continue supporting.

I was using them for FreeIPA server/client and awx.

**Comment by tteck** [comment, marked as answer]
Use the "All Templates LXC" script to create a Rocky Linux LXC

**Reply by amlucent** [reply]
thank you sir

**Comment by amlucent** [comment]
Im not seeing an option to make it a privileged container so I can do an nfs mount?

**Reply by tteck** [reply]
Backup, then restore privileged.
